client/src/components/organisms/RequestGroupForm.tsx

```diff
@@ -264,19 +265,26 @@ const RequestGroupForm: FunctionComponent<Props> = (props: Props) => {
     /* Functions for RequestGroup's Image */
     const updateImageError = (image: string) => {
         let error = "";
-        if (image === "") {
+        if (image === "" && uploadedImg === "") {
             error = "Please select an image";
         }
 
         setImageError(error);
         return error;
     };
 
+    const onUploadImg = (uploadedImg: string) => {
+        setChangeMade(true);
+        updateImageError(uploadedImg);
+        setUploadedImg(uploadedImg);
+    };
+
     const onImageChange = (newImage: string) => {
         setChangeMade(true);
         updateImageError(newImage);
-        if (images.find((imageUrl) => imageUrl === newImage)) {
+        if (newImage.length === 0 || images.find((imageUrl) => imageUrl === newImage)) {
```

Comment:
I think i'm missing something :(((
Why do we have `newImage.length === 0`?

Reply:
this is just a guess, but maybe it's for clearing the image? In `ImagePicker` the function `clearImage()` calls `onImageChange("")` so it might be to handle that case

Reply:
yeah I put that there so you could clear the image.